components/Onboarding/ConnectViaWallet/ConnectViaWalletTableViewItems.tsx

```diff
@@ -131,24 +133,27 @@ export const InstalledWalletsTableView = memo(
               let walletAddress: string = "";
 
               // Specific flow for Coinbase Wallet
-              if (wallet.name === "Coinbase Wallet") {
-                const wallet = await thirdwebConnect(async () => {
-                  const coinbaseWallet = createWallet("com.coinbase.wallet", {
-                    appMetadata: config.walletConnectConfig.appMetadata,
-                    mobileConfig: {
-                      callbackURL: `https://${config.websiteDomain}/coinbase`,
-                    },
-                  });
+              if (wallet.thirdwebId === "com.coinbase.wallet") {
+                // @todo => this is a hack to remove the smart wallet key from AsyncStorage
+                // because it's not being removed by the wallet itself
```

Comment:
Do other wallets remove their keys from storage?

Reply:
actually fixed by changing the way we disconnect thirdweb before laucnhing the new connection flow!
commit https://github.com/ephemeraHQ/converse-app/commit/2df37ea85c584a46c82cd01c0c81892c9a2b5ab7